### PR TITLE
Update features.md newUser snippets

### PR DESCRIPTION
**Pull request by vincanger**
more explicit examples

# Description

Vince's first PR. 
He's not sure if he's doing it right, but he figures he'd just give it a try. Give him a break -- he's trying to reach Waspeteer Level 4 🐝🐝🐝🐝

Fixes # (issue)

## Type of change

Please select the option(s) that is more relevant.

- [ ] Code cleanup
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: web/docs/language/features.md [hunk 1]
```diff
@@ -861,28 +861,35 @@ If you require more control in your authentication flow, you can achieve that in
...
+  // Your custom code before sign-up.
+  // ...
 
-    // Your custom code after sign-up.
-    // ...
-    return newUser
+  const newUser = context.entities.User.create({
+    data: { 
+      username: args.username, 
+      password: args.password // password hashed automatically by Wasp! 🐝
```
Comment (matijaSos) [inline web/docs/language/features.md:870]: nice! :D

File: web/docs/language/features.md [hunk 2]
```diff
@@ -861,28 +861,35 @@ If you require more control in your authentication flow, you can achieve that in
...
+
+  // Your custom code after sign-up.
+  // ...
+  return newUser
 }
 ```
 
 :::info
-You don't need to worry about hashing the password yourself! Even when you are using Prisma's client directly and calling `create()` with a plain-text password, Wasp put middleware in place that takes care of hashing it before storing it to the database. An additional middleware also performs field validation.
+You don't need to worry about hashing the password yourself! Even when you are using Prisma's client directly and calling `create()` with a plain-text password, Wasp's middleware takes care of hashing it before storing it in the database. An additional middleware also performs field validation.
```
Comment (matijaSos) [inline web/docs/language/features.md:881]: 👍

**Review by matijaSos — APPROVED** [review]
woohoo, your first PR, well done! Level 4 Waspeteer is imminent :D 🐝 🐝 🐝 🐝

**Comment by Martinsos** [comment]
Yay first PR @vincanger :D !
You did everything right, the only question is where to target this PR -> to `main` or to `release` branch. If you target it to `main` branch, it will get published once we release the new version (in 3 weeks).
If you instead target `release`, it can get published immediately.
I think in this case it is not urgent so `main` can also be ok, I am just saying this for future reference. You can also read more about his here: https://github.com/wasp-lang/wasp/tree/main/waspc#branching-and-merging-strategy .

**Comment by vincanger** [comment]
@Martinsos nice! Good to know. And, yes, I think `main` branch is fine as well

**Comment by matijaSos** [comment]
merged (I cherry-picked it to `release`)!

**Pull request closed without merging**